docs/getting-started/installing-shoko-server.mdx

```diff
@@ -126,5 +157,19 @@ Open Docker from the applications menu, then go to Containers. You'll need to do
 
 Please note that if you want to edit volumes or environment variables, you'll need to stop the container first.
 
-### Next Step
+## Docker (TrueNAS)
+
+### TrueNAS Scale
+
+TBD
+
+### TrueNAS Core
+
+Docker is not supported on this platform, as **TrueNAS Core** uses FreeBSD as base OS.
+
+## Docker (Unraid)
+
+TBD
+
+## Next Step
```

Comment:
Missing newline after the header. Mostly for ascetics. You can ignore this comment if we don't care for that.